apps/web/components/v2/apps/App.tsx

```diff
@@ -0,0 +1,337 @@
+import classNames from "classnames";
+import Link from "next/link";
+import React, { useEffect, useState } from "react";
+
+import useAddAppMutation from "@calcom/app-store/_utils/useAddAppMutation";
+import { InstallAppButton } from "@calcom/app-store/components";
+import LicenseRequired from "@calcom/features/ee/common/components/LicenseRequired";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import showToast from "@calcom/lib/notification";
+import { trpc } from "@calcom/trpc/react";
+import { App as AppType } from "@calcom/types/App";
+import Badge from "@calcom/ui/Badge";
+import { Icon } from "@calcom/ui/Icon";
+import { Button, SkeletonButton, Shell } from "@calcom/ui/v2";
+
+const Component = ({
+  name,
+  type,
+  logo,
+  body,
+  categories,
+  author,
+  price = 0,
+  commission,
+  isGlobal = false,
+  feeType,
+  docs,
+  website,
+  email,
+  tos,
+  privacy,
+  isProOnly,
+  images,
+}: Parameters<typeof App>[0]) => {
+  const { t } = useLocale();
+  const { data: user } = trpc.useQuery(["viewer.me"]);
+  const hasImages = images && images.length > 0;
+
+  const mutation = useAddAppMutation(null, {
+    onSuccess: () => {
+      showToast("App successfully installed", "success");
+    },
+    onError: (error) => {
+      if (error instanceof Error) showToast(error.message || "App could not be installed", "error");
+    },
+  });
+
+  const priceInDollar = Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "USD",
+    useGrouping: false,
+  }).format(price);
+  const [installedAppCount, setInstalledAppCount] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    async function getInstalledApp(appCredentialType: string) {
+      const queryParam = new URLSearchParams();
+      queryParam.set("app-credential-type", appCredentialType);
+      try {
+        const result = await fetch(`/api/app-store/installed?${queryParam.toString()}`, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        }).then((data) => {
+          setIsLoading(false);
+          return data;
+        });
+        if (result.status === 200) {
+          const res = await result.json();
+          setInstalledAppCount(res.count);
+        }
+      } catch (error) {
+        if (error instanceof Error) {
+          console.log(error.message);
+        }
+      }
+    }
+    getInstalledApp(type);
+  }, [type]);
+  const allowedMultipleInstalls = categories.indexOf("calendar") > -1;
+
+  return (
+    <>
+      <div className="px-4 pb-3 pt-3 md:px-8 lg:px-0 lg:pt-0">
+        <Link href="/apps">
+          <a className="text-md hover:text-brand-400 inline-flex rounded-sm py-2 font-semibold  text-gray-900">
+            <Icon.FiArrowLeft className="mr-2 h-6 w-6" /> {t("app_store")}
```

Comment:
Not sure your thoughts @Jaibles but having the "<- App Store" back button scroll with the screenshots looks weird to me, maybe is just a NIT. The designs doesn't show the desired behavior here. And when the text side is long enough, scrolling phases come up, see: 

https://user-images.githubusercontent.com/467258/187283383-b75e0e28-e5c0-486a-95d2-fa07893e55c9.mov